static/index.html

```diff
@@ -503,6 +503,10 @@ <h3 class="mb-5">Other switches</h3>
                 } else {
                   _self.error = e;
                 }
+
+                if (e.error === 'login_required') {
+                  _self.isAuthenticated = false;
+                }
```

Comment:
This catches the `login_required` in our playground, allowing us to reset the authentication state when needed based on the response from Auth0.